docs/APISocketOptions.md

```diff
@@ -0,0 +1,1511 @@
+| OptName              | Since | Binding | Type       |  Units  | Default  | Range  | Dir | Entity |
+| -------------------- | ----- | ------- | ---------- | ------- | -------- | ------ | --- | ------ |
+| `SRTO_MSS`           |       | pre     | `int32_t`  | bytes   | 1500     | 76..   | RW  | GSD    |
+
+- Maximum Segment Size. Used for buffer allocation and rate calculation using 
+packet counter assuming fully filled packets. Each party can set its own MSS 
+value independently. During a handshake the parties exchange MSS values, and 
+the lowest is used.
+
+*Generally on the internet MSS is 1500 by default. This is the maximum 
+size of a UDP packet and can be only decreased, unless you have some unusual 
+dedicated network settings. MSS is not to be confused with the size of the UDP 
+payload or SRT payload - this size is the size of the IP packet, including the 
+UDP and SRT headers* 
+
+
+[Return to list](#list-of-options)
+
+
+
+#### SRTO_NAKREPORT
+
+| OptName              | Since | Binding | Type       |  Units  | Default  | Range  | Dir | Entity |
+| -------------------- | ----- | ------- | ---------- | ------- | -------- | ------ | --- | ------ |
+| `SRTO_NAKREPORT`     | 1.1.0 | pre     | `bool`     |         |  *       |        | RW  | GSD+   |
+
+- When set to true, every report for a detected loss will be repeated when the
+timeout for the expected retransmission of this loss has expired and the
+missing packet still wasn't recovered, or wasn't conditionally dropped (see
+[`SRTO_TLPKTDROP`](#SRTO_TLPKTDROP)).
+
+- The default is true for Live mode, and false for File mode (see [`SRTO_TRANSTYPE`](#SRTO_TRANSTYPE)).
+
+
+[Return to list](#list-of-options)
+
+
+
+#### SRTO_OHEADBW
+
+| OptName              | Since | Binding | Type       |  Units  | Default  | Range  | Dir | Entity |
+| -------------------- | ----- | ------- | ---------- | ------- | -------- | ------ | --- | ------ |
+| `SRTO_OHEADBW`       | 1.0.5 | post    | `int32_t`  | %       | 25       | 5..100 | W   | GSD    |
+
+- Recovery bandwidth overhead above input rate (see `[`SRTO_INPUTBW`](#SRTO_INPUTBW)`), 
+in percentage of the input rate. It is effective only if `SRTO_MAXBW` is set to 0.
+
+- *Sender: user configurable, default: 25%.* 
+
+- Recommendations:
+
+	- *Overhead is intended to give you extra bandwidth for the case when a packet 
+	has taken part of the bandwidth, but then was lost and has to be retransmitted. 
+	Therefore the effective maximum bandwidth should be appropriately higher than 
+	your stream's bitrate so that there's some room for retransmission, but still 
+	limited so that the retransmitted packets don't cause the bandwidth usage to 
+	skyrocket when larger groups of packets are lost*
+
+	- *Don't configure it too low and avoid 0 in the case when you have the 
+	`SRTO_INPUTBW` option set to 0 (automatic). Otherwise your stream will choke 
+	and break quickly at any rise in packet loss.*
+
+- ***To do: set-only; get should be supported.***
```

Comment:
@ethouris There is another "to do" here.

Reply:
Preserve this. There's another PR from Jean that introduced the new `SRTO_OUTPACEMODE` option to replace the bandwidth limitation settings with a unified option, this will be an opportunity to provide somehow the possibility of reading the value of the associated options.